interop-tests/Cargo.toml

```
@@ -0,0 +1,38 @@
+[package]
+name = "forest-interop-tests"
+version = "0.1.0"
+authors = ["ChainSafe Systems <info@chainsafe.io>"]
```

It's likely a better point of contact.

```suggestion
authors = ["ChainSafe Systems <forest@chainsafe.io>"]
```